.github/workflows/aws-terraform.yml

```diff
@@ -56,13 +55,23 @@ env:
 jobs:
   aws-terraform-env:
     name: Aws Environment Terraform
-    runs-on: ubuntu-22.04
+    runs-on: ${{ vars.RUNNER_OS }}
```

Comment:
how come? whats wrong with using 22.04

Reply:
RUNNER_OS (Repository variable) value is set to `ubuntu-22.04`